### PR TITLE
Add the double-barrel four winds shotgun and update the shotgun trap.

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

SUMMARY: Content "Add the double-barrel four winds shotgun and update the shotgun trap"
#### Purpose of change

The double-barrel version of the four winds shotgun is here! Plus, a change to the shotgun traps that will finally make then semi-viable.
#### Describe the solution

This adds the awaited and promised double barrel four wind shotgun. With this comes a quick change to the shotgun trap.

#### Describe alternatives you've considered

None!
#### Testing

Add changes.
Summoned a gun trap.
Blew apart my torso.
DIsarm gun trap.

#### Additional context

Probable weapon update: the Powerhead! Its niche is that it may be used underwater or on land!

### CODE REVIEW

File: data/json/items/gun/shot.json [hunk 1]
```diff
@@ -848,5 +848,20 @@
...
+    "type": "GUN",
+    "name": { "str": "double four winds shotgun" },
+    "description": "A crude shotgun, composed of four thick steel pipes, two end caps and two nails.  The lack of sights make this weapon only useful at point-blank range.",
+    "weight": "7260 g",
+    "volume": "4530 ml",
+    "dispersion": 960,
+    "durability": 4,
+    "clip_size": 2,
+    "reload": 900,
+    "modes": [ [ "DEFAULT", "double", 2 ] ], //no provision for seperate triggers or ANYTHING
```
Comment (mlangsdorf) [inline data/json/items/gun/shot.json:864]: That is not how you do comments in JSON; it should be 
`"//": "no provision for seperate triggers or ANYTHING",
```

**Comment by Xenomorph-III** [comment]
Just wanted it make it easier to obtain. And I was discussing it on Reddit with someone, saying that it would be a good use for it seeing it's effective range is point blank.

File: data/json/items/gun/shot.json (hunk 1 above)
Comment (Xenomorph-III) [reply data/json/items/gun/shot.json]: It's fixed now.

File: data/json/items/migration.json [hunk 2]
```diff
@@ -1235,8 +1235,13 @@
...
-    "id": "slam_shotgun",
+    "id": "four_winds_shotgun",
     "type": "MIGRATION",
-    "replace": "four_winds_shotgun"
+    "replace": "slamfire_shotgun"
+  },
+  {
+    "id": "four_winds_shotgun_d",
+    "type": "MIGRATION",
+    "replace": "slamfire_shotgun_d"
```
Comment (LaVeyanFiend) [inline data/json/items/migration.json:1245]: You never made four_winds_shotgun_d, so this entry is causing the error.
```suggestion

```

**Comment by I-am-Erk** [comment]
It sounds like a better fix for the shotgun rarity problem might be to make double barreled shotguns less rare.

**Pull request merged**